mopidy/core/actor.py

```diff
@@ -136,6 +140,103 @@ def tags_changed(self, tags):
             self.playback._stream_title = title
             CoreListener.send('stream_title_changed', title=title)
 
+    def setup(self):
```

Comment:
`setup`and `teardown` are not in the  JSON_RPC.
Calling them twice will not hurt (with the current implementation). I think we don't need the if-already-called check here (now).